OBJECT/DoorSystemSetDoorState.md

```diff
@@ -6,19 +6,23 @@ aliases: ["0x6BAB9442830C7F53","_SET_DOOR_ACCELERATION_LIMIT"]
 
 ```c
 // 0x6BAB9442830C7F53 0xDF83DB47
-void DOOR_SYSTEM_SET_DOOR_STATE(Hash doorHash, int limit, BOOL p2, BOOL p3);
+void DOOR_SYSTEM_SET_DOOR_STATE(Hash doorHash, int state, BOOL requestDoor, BOOL forceUpdate);
 ```
 
-```
-Sets the acceleration limit of a door.
-How fast it can open, or the inverse hinge resistance.
-A limit of 0 seems to lock doors.
-p2 is always 0, p3 is always 1.
-```
+Lockstates not applied and CNetObjDoor's not created until [DOOR_SYSTEM_GET_IS_PHYSICS_LOADED](#_0xDF97CDD4FC08FD34) returns true.
+
+### Door lock states: (v323)
```

Comment:
Are sub-headings read by the parser and placed in description?

Reply:
For reference: https://runtime.fivem.net/doc/natives/?_0x262B14F48D29DE80

NativesFive.cs
```cs
/// This native is used to set component variation on a ped. Components, drawables and textures IDs are related to the ped model.
/// ### MP Freemode list of components
/// **0**: Face
/// **1**: Mask
/// **2**: Hair
/// **3**: Torso
/// **4**: Leg
/// **5**: Parachute / bag
/// **6**: Shoes
/// **7**: Accessory
/// **8**: Undershirt
/// **9**: Kevlar
/// **10**: Badge
/// **11**: Torso 2
/// ### Related and useful natives
/// [GET_NUMBER_OF_PED_DRAWABLE_VARIATIONS](#_0x27561561732A7842)
/// [GET_NUMBER_OF_PED_TEXTURE_VARIATIONS](#_0x8F7156A3142A6BAD)
/// [List of component/props ID](gtaxscripting.blogspot.com/2016/04/gta-v-peds-component-and-props.html) of player_two with examples
```

natives_universal.lua
```lua
-- ### MP Freemode list of components
-- **0**: Face
-- **1**: Mask
-- **2**: Hair
-- **3**: Torso
-- **4**: Leg
-- **5**: Parachute / bag
-- **6**: Shoes
-- **7**: Accessory
-- **8**: Undershirt
-- **9**: Kevlar
-- **10**: Badge
-- **11**: Torso 2
-- ### Related and useful natives
-- [GET_NUMBER_OF_PED_DRAWABLE_VARIATIONS](#_0x27561561732A7842)
-- [GET_NUMBER_OF_PED_TEXTURE_VARIATIONS](#_0x8F7156A3142A6BAD)
-- [List of component/props ID](gtaxscripting.blogspot.com/2016/04/gta-
```

I'm not entirely sure how documentation for NativesFive.cs renders in VisualStudio. However, I was attempting to be consistent with what already exists.